frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -665,14 +665,18 @@ class Question {
     return question;
   }
 
-  parameters({ collectionPreview } = {}): ParameterObject[] {
+  private _getParameters = _.memoize((collectionPreview: boolean) => {
```

Comment:
I think this is dangerous: because `_.memoize` does not receive most of it's values (like `this._parameterValues`) as an argument and so will not be part of `memoize`'s cache key.


I know `this._parameterValues` is never set in the `Question` class but that's not a guarantee.

Reply:
@romeovs `Question` is immutable, we don't need to pass options to memoize.